set.mm

```diff
@@ -498243,6 +498422,29 @@ then this is also a division ring (see ~ fldgensdrg ).  If the base
   $}
 
 
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+  Ideals
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+$)
+
+  ${
+    2idllidld.1 $e |- ( ph -> I e. ( 2Ideal ` R ) ) $.
+    $( A two-sided ideal is a left ideal.  (Contributed by Thierry Arnoux,
+       9-Mar-2025.) $)
+    2idllidld $p |- ( ph -> I e. ( LIdeal ` R ) ) $=
+      ( clidl cfv coppr c2idl cin eqid 2idlval eleqtrdi elin1d ) ABEFZBGFZEFZCA
+      CBHFZNPIDBQNPONJOJPJQJKLM $.
+
+    2idllidld.o $e |- O = ( oppR ` R ) $.
+    $( A two-sided ideal is a right ideal.  (Contributed by Thierry Arnoux,
+       9-Mar-2025.) $)
+    2idlridld $p |- ( ph -> I e. ( LIdeal ` O ) ) $=
+      ( clidl cfv c2idl cin eqid 2idlval eleqtrdi elin2d ) ABGHZDGHZCACBIHZOPJE
+      BQOPDOKFPKQKLMN $.
+  $}
+
+
```

Comment:
These theorems should be moved to main, so that I can use them, too.

Reply:
... and the following ~qusmul2, too!

Reply:
Usually we move them to main in the same PR where we use them, but if you prefer I could move them in this PR?

Reply:
They seem useful enough to be in main

Reply:
> Usually we move them to main in the same PR where we use them, but if you prefer I could move them in this PR?

Yes, please.

Reply:
OK! Done in https://github.com/metamath/set.mm/pull/4702/commits/9d63a4d7a9982b3829be673dac2a3babfa94ed23
I did not mention it in `change-set.txt` because those theorems were new in my mathbox anyway.